mix.exs

```diff
@@ -39,8 +39,8 @@ defmodule PhoenixHtmlSanitizer.Mixfile do
   # Type `mix help deps` for more examples and options
   defp deps do
     [
-      {:phoenix_html, "~> 2.0 or ~> 3.0"},
-      {:html_sanitize_ex, "~> 1.3.0"},
+      {:phoenix_html, "~> 4.1"},
```

Comment:
this may be able to support the old version simultaneously, but I'm trying to get it working with just the new version first